src/renderer/components/+catalog/catalog-entity-item.tsx

```diff
@@ -26,12 +26,9 @@ import type { ItemObject } from "../../../common/item.store";
 import { Badge } from "../badge";
 import { navigation } from "../../navigation";
 import { searchUrlParam } from "../input";
-import { makeCss } from "../../../common/utils/makeCss";
 import { KubeObject } from "../../../common/k8s-api/kube-object";
 import type { CatalogEntityRegistry } from "../../api/catalog-entity-registry";
 
-const css = makeCss(styles);
```

Comment:
To support css-classes auto-completion use [vscode-css-modules](https://marketplace.visualstudio.com/items?itemName=clinyong.vscode-css-modules) plugin.